todo_list.py

```diff
@@ -1,10 +1,24 @@
-list = [{"id": 10, "desc": "wash car"}]
+list = []
+
+def add_todo(description):
+    list.append({"id": len(list) + 1, "desc": description})
 
 # While loop set to True
 # Break when user wants to quit the app
-if len(list) == 0:
-    print("Todo list is empty!")
-else:
-    for i in range(len(list)):
-        print("[ ]", list[i]["id"], "-", list[i]["desc"])
-
+while True:
+    if len(list) == 0:
+        print("Todo list is empty!")
+    else:
+        for i in range(len(list)):
+            print("[ ]", list[i]["id"], "-", list[i]["desc"])
```

Comment:
Todo status should be part of the todo object pushed to the list. It has no use being hardcoded.